.github/workflows/main.yml

```diff
@@ -217,7 +217,14 @@ jobs:
           source "$CONDA/etc/profile.d/conda.sh"
           conda activate psi4env
           make mypy
```

Comment:
I imagine you could have done a simple if else around the make mypy to do it one way or another depending on whether python version was 3.7 or not - akin to what is done in line 176 above. But this does the job too.